.travis.yml

```diff
@@ -4,22 +4,19 @@ node_js:
   - 6
   - 8
   - 9
+env:
+  - PYTHON_VERSION=2.7.14
+  - PYTHON_VERSION=3.6.3
 cache:
   yarn: true
   directories:
   - node_modules
-  - /home/travis/.pyenv_cache
+before_install:
+  - pyenv install -s $PYTHON_VERSION
+  - pyenv global $PYTHON_VERSION
+  - python --version
 install:
   - yarn install
-before_install:
-  - export PYTHON_BUILD_CACHE_PATH="/home/travis/.pyenv_cache"
-  - curl -L https://raw.githubusercontent.com/pyenv/pyenv-installer/master/bin/pyenv-installer | bash
```

Comment:
Turns out pyenv is already available on Travis (even when the language is `node_js`), and they have a decent selection of versions already installed.